### PR TITLE
 [Doc] PR for Platform "Connect to a database node" - some steps and screenshots are outdated (Doc9400) 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-10165--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/create-deployments/create-universe-multi-zone/#examine-the-universe

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md [hunk 1]
```diff
@@ -61,31 +61,17 @@ The following illustration shows a newly-created universe in Pending state:
...
-
-![Instances for a Pending Universe](/images/ee/multi-zone-universe-gcp-instances.png)
+The **Universes** view allows you to examine various aspects of the universe:
+
+- **Overview** provides the information on the current Yugabyte Platform version, the number of nodes included in the primary cluster, the cost associated with running the universe, the CPU and disk usage, the geographical location of the nodes, the operations per second and average latency, the number of different types of tables, as well as the health monitor.
+- **Tables** provides details about YSQL, YCQL, and YEDIS tables included in the universe.
+- **Nodes** provide details on nodes included in the universe and allows to perform actions on a specific node (connect, stop, remove, display live and slow queries, download logs). You can also use **Nodes** to open the cloud provider's instances page. For example, in case of GCP, if you navigate to **Compute Engine > VM Instances** and search for instances that contain the name of your universe in the instances name, you should see a list of instances.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:68]: allows you to perform actions

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md [hunk 2]
```diff
@@ -61,31 +61,17 @@ The following illustration shows a newly-created universe in Pending state:
...
+- **Metrics** displays graphs representing information on operations, latency, and other parameter for each type of node and server.
+- **Queries** displays details about live and slow queries that you can filter by column and text.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:70]: parameters

**Review by ddhodge — APPROVED** [review]
minor fixes otherwise LGTM

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:68]: Fixed

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:70]: Fixed

**Pull request merged**